Source/src/utils/UUID.h

```diff
@@ -11,7 +11,7 @@
 
 namespace Hachiko
 {
-    using UID = unsigned long long; //64 bits positive number (size_t)
+    typedef unsigned long long UID; //64 bits positive number (size_t)
```

Comment:
is there a particular reason to change this? typedef won't work with templates, according to my research

Reply:
I had problems and i tried changing it, we can go back to using no probs